src/containers/LoginModal/utils.ts

```diff
@@ -46,6 +49,8 @@ export function toProviderType(
   modalOptionType: LoginModalOptionType
 ): ProviderType {
   switch (modalOptionType) {
+    case METAMASK_MOBILE:
```

Comment:
let's add a comment here saying that we're adding METAMASK_MOBILE with the same WALLET_CONNECT logic, so if we come back later to this code we know why we put it like this :)